packages/libpngwriter/CMakeLists.txt.patch

```diff
@@ -0,0 +1,36 @@
+diff -uNr pngwriter-0.7.0/CMakeLists.txt pngwriter-0.7.0.mod/CMakeLists.txt
+--- pngwriter-0.7.0/CMakeLists.txt	2018-01-22 21:10:17.000000000 +0800
++++ pngwriter-0.7.0.mod/CMakeLists.txt	2022-05-31 12:29:03.197155948 +0800
+@@ -70,13 +70,23 @@
+ 
+ # Targets #####################################################################
+ #
++
++INCLUDE_DIRECTORIES(@TERMUX_PREFIX@/include/freetype2)
++
+ add_library(PNGwriter
+     src/pngwriter.cc
+     src/pngwriter.h
+ )
+ 
++add_library(PNGwriter_shared SHARED
++	src/pngwriter.cc
++	src/pngwriter.h
++)
++
+ target_link_libraries(PNGwriter PUBLIC PNG::PNG)
+ target_link_libraries(PNGwriter PUBLIC ZLIB::ZLIB)
++target_link_libraries(PNGwriter_shared PUBLIC PNG::PNG)
++target_link_libraries(PNGwriter_shared PUBLIC ZLIB::ZLIB)
+ if(NOT WIN32)
+     # automatically added on windows
+     target_link_libraries(PNGwriter PRIVATE m)
```

Comment:
I would guess that the shared library needs to be linked against libm here as well

Reply:
![image](https://user-images.githubusercontent.com/57583560/171586512-4c56713a-114f-4813-b2e9-ca694d4f7231.png)